database/generateData.js

```diff
@@ -0,0 +1,97 @@
+      const includesList = ['drinks', 'food', 'equipment'];
+      const cuisineList = ['Japanese', 'Chinese', 'Brazilian', 'American'];
+      const videoUrlList = ['https://emerald.org', 'http://jerome.com', 'http://estella.com', 'http://alexa.net', 'https://mitchell.com'];
+      const languagesList = ['Japanese', 'Chinese', 'Portuguese', 'English', 'German', 'French'];
+
+      const experience = {
+        experienceId: id,
+        title: titleList[Math.floor(Math.random() * titleList.length)],
+        city: cityList[Math.floor(Math.random() * cityList.length)],
+        state: stateList[Math.floor(Math.random() * stateList.length)],
+        country: countryList[Math.floor(Math.random() * countryList.length)],
+        category: categoryList[Math.floor(Math.random() * categoryList.length)],
+        activity: activityList[Math.floor(Math.random() * activityList.length)],
+        averageRating: Math.floor(Math.random() * (5 - 1) + 1),
+        numberOfReviews: Math.floor(Math.random() * (100 - 1) + 1),
+        duration: Math.floor(Math.random() * (6 - 1) + 1),
+        groupSize: Math.floor(Math.random() * (40 - 1) + 1),
+        includes: [],
+        cuisine: cuisineList[Math.floor(Math.random() * cuisineList.length)],
+        hostedLanguages: [],
+        costPerPerson: Math.floor(Math.random() * (300 - 10) + 10),
+        imageUrls: [],
+        videoUrl: videoUrlList[Math.floor(Math.random() * videoUrlList.length)],
+      };
+
+      // add to includes array
+      const numIncludes = Math.floor(Math.random() * 4 + 1);
+      for (let j = numIncludes; j > 0; j -= 1) {
+        const randomWord = includesList[Math.floor(Math.random() * includesList.length)];
+        experience.includes.push(randomWord);
+      }
+
+      // add to hostedLanguages
+      const numLanguages = Math.floor(Math.random() * 4 + 1);
+      for (let k = numLanguages; k > 0; k -= 1) {
+        const randomWord = languagesList[Math.floor(Math.random() * languagesList.length)];
+        experience.hostedLanguages.push(randomWord);
+      }
+
+      // add 5 photos
+      for (let m = 5; m > 0; m -= 1) {
+        const randomNum = Math.floor(Math.random() * 20 + 1);
+        // need to connect to aws s3 for the url
+        const url = `https://fec-media.s3.amazonaws.com/photo${randomNum}.jpg`;
+        experience.imageUrls.push(url);
+      }
+
+      const includes = `"{${experience.includes}}"`;
+      const hostedLanguages = `"{${experience.hostedLanguages}}"`;
+      const imageUrls = `"{${experience.imageUrls}}"`;
+
+      const data = `${experience.experienceId},${experience.title},${experience.city},${experience.state},${experience.country},${experience.category},${experience.activity},${experience.averageRating},${experience.numberOfReviews},${experience.duration},${experience.groupSize},${includes},${experience.cuisine},${hostedLanguages},${experience.costPerPerson},${imageUrls},${experience.videoUrl}\n`;
```

Comment:
Suggest breaking this single long line into multiple (one var per line.) Hard to read.

Reply:
Thanks @Kraig5.